index.js

```diff
@@ -1,32 +1,44 @@
+function onDocumentReady(fn) {
+    if (document.readyState !== 'loading') {
+        fn();
+    } else {
+        document.addEventListener('DOMContentLoaded', fn);
+    }
+}
+
 if (/^https:\/\/(.*)?\.atlassian\.net\/secure\/RapidBoard.jspa\?rapidView/gmi.test(window.location.href)) {
     const PAGE_UP = 33;
     const PAGE_DOWN = 34;
     const ARROW_UP = 38;
     const ARROW_DOWN = 40;
 
-    $(document).ready(() => {
-        $('.ghx-swimlane').not('.ghx-closed').each((_, elem) => $(elem).addClass('ghx-closed'));
-        $('.ghx-swimlane').last().addClass('ghx-last')
-        $('.ghx-swimlane.ghx-first').removeClass('ghx-closed');
-	
-        $("body, div#content, div#ghx-pool").keydown(({ keyCode, preventDefault }) => {
-            preventDefault();
+    onDocumentReady(() => {
+        const swimlanes = Array.from(document.querySelectorAll('.ghx-swimlane'));
+
+        swimlanes.forEach(elem => elem.classList.add('ghx-closed'));
+        swimlanes.slice(-1)[0].classList.add('ghx-last');
```

Comment:
I think that `swimlanes` should be treated as an immutable. By using `.splice` you are mutating the array.

You could use some uber cool deconstruction to get the last entry of `swimlanes`

```suggestion
const [, last] = swimlanes;
last.classList.add('ghx-last');
```

Reply:
@arelstone I think you misread my code, I used the method `.slice` which does not mutate the array.

Reply:
My bad.. Sorry